DesignSafe-CI/portal
# Replace Tapis downloads with the streaming zip service.

## Overview: ##
Points the 

## PR Status: ##


## Related Jira tickets: ##

* [DES-1889](https://jira.tacc.utexas.edu/browse/DES-1889)

## Summary of Changes: ##

## Testing Steps: ##
1. Select 2 files in the Data Depot and hit the download button. You should see a ZIP archive begin to download.
2. Select a folder in the Data Depot and hit the download button. This should also trigger a ZIP download.
3. Select a single file and make sure that the download button still works as intended.
